### ISSUE TITLE
Editing an old task without checklist brings the "Checklist" title

### ISSUE THREAD

**Issue by ghost** [HabitRPG/habitica#4926]
While editing an old task, I noticed something unusual:
![screenshot - 26032015 - 20 31 57](https://cloud.githubusercontent.com/assets/4882150/6856501/bd1975b2-d400-11e4-96e7-1a940d81ff98.png)
Although the task has no checklist, the title _Checklist_ appears under the _Add Checklist_ button, and then a horizontal rule, as if it is expecting a checklist to appear next.

So I tried to create a new task, and editing it, to see if it was the same.
![screenshot - 26032015 - 20 46 58](https://cloud.githubusercontent.com/assets/4882150/6856555/16e9a94a-d401-11e4-9d33-2878b63452f4.png)
There, the _Checklist_ title does not appear. I tried saving the task, and editing it again, adding tags, ... The only way I could make the _Checklist_ title appear is to click on _Add Checklist_ and save without adding anything. Still, I'm pretty sure I did not do that on all of my older tasks, so there's got to be some other trigger.

I asked a friend of mine to check on her tasks, and it was exactly the same behaviour. By the way, it appears for either dailies or todos. I'm pretty sure it can be reproduced on any old task without checklist. I created a new sample task, just to see if tomorrow brings the _Checklist_ title in it without doing anything.

Although this behaviour is not critical at all, it adds a lot of vertical space in each corresponding task, which is a lot when using lower screen resolution (as I am on my old 1024x600 netbook).

If needed, I ran these tests in Firefox 36 in Xubuntu.

**Comment by Alys** [comment]
Good catch! Thank you.

We'll be changing the way the checklist button, title, and info tip are presented (see https://github.com/HabitRPG/habitrpg/pull/4821#issuecomment-78367808 and later comments), so I'm going to put this issue on hold until after that. Whatever weird bug is causing this might be fixed by that change, but even if not, we don't want the two fixes conflicting. I've made a To-Do for myself to ensure that this issue gets taken off hold once #4821 is merged.

**Comment by ghost** [comment]
I think I nailed the trigger. Although opening an empty checklist does bring the title, I still thought that this wasn't done that way with older tasks. I first thought that it had something to do with cron, but after a simple refresh of the page, now the task I created 2 hours ago show this behaviour. I tested it with a newly created task. A refresh immediately brings the _Checklist_ title.

**Comment by Alys** [comment]
I'm closing this because it's been fixed in one of the PRs that was merged today. The fix will appear on the production site within a few days.
